bonnie-rest/src/main/java/com/cgi/bonnie/bonnierest/OrderController.java

```diff
@@ -53,6 +53,7 @@ public ResponseEntity<List<Order>> getAllOrders() {
     public ResponseEntity<String> createOrder(@RequestBody OrderRequest request) {
         try {
             Status status = Status.valueOf(request.getStatus().toUpperCase());
+//fixme order without shopOrderId, placementDate and metadata
```

Comment:
Maybe we should check these?

Reply:
in this version the OrderRequest also doesn't content some mandatory field. Those have to implement first than can use createOrder( OrderRequest or Order )